pipeline/cip_crosswalk.py

```diff
@@ -287,7 +287,7 @@ class NCSESClassifier:
             FieldTaxonomy.ncses_detailed_field_group: NCSESDetailedFieldGroup.ag_sci.value,
         },
         "15.1799": {
-            "ncses_cip_string": "Energy Systems Technologies/Technicials, Other",
+            "ncses_cip_string": "Energy Systems Technologies/Technicals, Other",
```

Comment:
This should be `Energy Systems Technologies/Technicians, Other`

https://nces.ed.gov/ipeds/cipcode/cipdetail.aspx?y=56&cipid=91827